src/components/code-editor/CodeEditor.vue

```diff
@@ -104,6 +105,13 @@ export default {
                 }
             });
         },
+        handleError(annotations) {
+            // Filter annotations with type 'error'
+            const errorAnnotations = annotations.filter(annotation => annotation.type === "error");
+
+            // Emit 'onError' event with the array of error annotations
+            this.$emit("onError", errorAnnotations);
```

Comment:
The `onError` event should only be emitted if there are errors

Reply:
will add the check for it back @AbdAsh

Reply:
sent replay regarding this on slack